tests/conftest.py

```diff
@@ -78,6 +79,21 @@ def labels_relative_path() -> os.PathLike:
     return ASSETS_DIR / "labels.csv"
 
 
+def labels_n_classes_df(n_classes):
+    """Get up a labels data frame where the labels are up to
+    26 classes.
+    """
+    if n_classes > len(string.ascii_uppercase):
+        raise ValueError("n_classes must be less than 26")
```

Comment:
what's the point of this?

Reply:
just cuz it's easy to implement guaranteed unique classes by just using the uppercase letters.

Reply:
ah make sense